src/tribler/gui/widgets/downloadspage.py

```diff
@@ -459,12 +494,16 @@ def on_export_download(self, checked):
             self.dialog.show()
 
     def on_export_download_dialog_done(self, action):
+        def on_finish(result: Tuple):
+            data, _ = result
+            self.on_export_download_request_done(filename, data)
+
         selected_item = self.selected_items[:1]
         if action == 0 and selected_item:
             filename = self.dialog.dialog_widget.dialog_input.text()
-            TriblerFileDownloadRequest(
+            FileDownloadRequest(
                 f"downloads/{selected_item[0].download_info['infohash']}/torrent",
-                lambda data: self.on_export_download_request_done(filename, data),
+                on_finish,
             )
 
```

Comment:
It is not necessary to use a separate `FileDownloadRequest` class as the same can be expressed with the base Request object:
```python
request_manager.get(f"downloads/{selected_item[0].download_info['infohash']}/torrent",
                    on_finish, priority=QNetworkRequest.LowPriority, raw_response=True)

```